src/main/java/core/basesyntax/FileWork.java

```
@@ -8,7 +13,36 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final char CONDITION = 'w';
+    private static final String WHITESPACE = " ";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder inputText = new StringBuilder();
+        String inputLine = null;
+        try (BufferedReader br = new BufferedReader(new FileReader(fileName))) {
+            while (br.ready()) {
+                inputLine = br.readLine();
+                inputText.append(inputLine.toLowerCase()
+                        .replaceAll("\\p{Punct}", ""))
+                        .append(WHITESPACE);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

don't ignore exceptions

---

Done. Thank you!